test-it/GetObjPathArrayIndex.via

```diff
@@ -0,0 +1,61 @@
+// Autogenerated Vireo assembly file
+// Generated on Tue Oct 08 15:12:36 2013
+define(Point c(e(.Int32 x)e(.Int32 y)))
+//---------------------------------------------------
+//VI Definition: stringManipulation.vi
+define(videf1  dv(.VirtualInstrument (
```

Comment:
Ok, I had one in there but overwrote it with a different test case.  It returns default data for out of bounds.